pkg/log/handler_sentry.go

```diff
@@ -0,0 +1,82 @@
+package log
+
+import (
+	"fmt"
+	"github.com/applike/gosoline/pkg/cfg"
+	"github.com/getsentry/sentry-go"
+	"time"
+)
+
+func init() {
+	AddHandlerFactory("sentry", handlerSentryFactory)
+}
+
+func handlerSentryFactory(config cfg.Config, _ int) (Handler, error) {
+	return NewHandlerSentry(config)
+}
+
+type HandlerSentry struct {
+	hub *sentry.Hub
+}
+
+func NewHandlerSentry(config cfg.Config) (*HandlerSentry, error) {
+	env := config.GetString("env")
+	appName := config.GetString("app_name")
```

Comment:
At some point we should add methods for these operations to the config package, otherwise we will never be able to rename `app_name` should we eventually want to do that

Reply:
Yes, this has to be improved somewhen.. but's that's a big change by itself..